exporter/logzioexporter/README.md

```diff
@@ -0,0 +1,18 @@
+# Logzio Exporter
+
+This exporter supports sending trace data to [Logz.io](https://www.logz.io)
+
+The following configuration options are supported:
+
+* `account_token` (Required): Your logz.io account token for your tracing account.
```

Comment:
If this is required, would it be preferable to check for zero value in factory helper or additional validate() method?